playground/src/main/java/mx/yellowme/youst/playground/ui/camerax/CameraXActivity.kt

```diff
@@ -0,0 +1,78 @@
+package mx.yellowme.youst.playground.ui.camerax
+
+import android.Manifest
+import android.content.pm.PackageManager
+import android.os.Bundle
+import android.widget.Toast
+import androidx.core.app.ActivityCompat
+import androidx.core.content.ContextCompat
+import androidx.lifecycle.LifecycleOwner
+import kotlinx.android.synthetic.main.screen_camerax.*
+import mx.yellowme.youst.core.hooks.BaseActivity
+import mx.yellowme.youst.playground.R
+
+/**
+ * @author adrianleyvasanchez
+ * @since 10,December,2019
+ */
+class CameraXActivity : BaseActivity(), LifecycleOwner {
+
+    //region Attributes
+
+    override val layoutId = R.layout.screen_camerax
+
+    private val requestCodePermission = 10
+
+    private val requiredPermissions = arrayOf(Manifest.permission.CAMERA)
+
+    //endregion
+
+    //region Lifecycle
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        if (allPermissionsGranted()) {
+            initializeCamera()
+            captureButton.setOnClickListener {
+                cameraxy.captureImage()
+            }
+        } else {
+            ActivityCompat.requestPermissions(
+                this, requiredPermissions, requestCodePermission)
+        }
+    }
+
+    //endregion
+
+    //region Helpers
+
+    private fun initializeCamera() {
+        cameraxy.run {
```

Comment:
Si no accederás a mas propiedades del cameraxy que opinas de dejarlo como:

`cameraxy.lifecycleOwner = this`